15.md

```diff
@@ -75,21 +74,21 @@ Fields that are not self-explanatory:
 
 ```json
 {
-  "tags": [["d", <string, id of stall]],
+  "tags": [["d", <string, id of the stall>]],
   ...
 }
 ```
- - the `d` tag is required, its value MUST be the same as the stall `id`.
+ - the `d` tag is required and represents the stall id.
 
 ### Event `30018`: Create or update a product
 
 **Event Content**
 
 ```json
 {
-  "id": <string, id generated by the merchant (sequential ids are discouraged)>,
-  "stall_id": <string, id of the stall to which this product belong to>,
```

Comment:
should not be removed, see my https://github.com/nostr-protocol/nips/pull/1166#discussion_r1559095810

Reply:
Just [answered](https://github.com/nostr-protocol/nips/pull/1166#discussion_r1559206125)

Reply:
This is a nitpick improvement unrelated with the other changes in this PR.
Its a NACK from my side.

Reply:
I would say that there is no necessity to have that redundant information in the events, there is no separation of concerns or something around that, same information duplicated two times in the same event, from my point of view it just creates ambiguity that can lead to inconsistencies